### PR TITLE
added docs for 1.7.0 data-table changes

### PR DESCRIPTION
**Pull request by jonthans**
made example inserts into data-table docs consistent

### CODE REVIEW

**Comment by jonthans** [comment]
I'm not sure why this includes changes to content/resources/hexagon/1.6.0... I'm not even sure why those files are even included in the git repo?

File: content/modules/data-table/versions/1.7.0.um [hunk 1]
```diff
@@ -0,0 +1,26 @@
+@version 1.5.0
+  @examples
+    @example
+      @html: @inline 1.5.0/simple-example.html
```
Comment (jonthans) [inline content/modules/data-table/versions/1.7.0.um:4]: I guess as examples can become out of date we'll need to repeat them in later versions if we want to add more examples. Just makes it a bit more tedious

Comment (charlieTheBotDev) [reply content/modules/data-table/versions/1.7.0.um:4]: `@version 1.5.0` used in file `1.7.0.um`?

Comment (jonthans) [reply content/modules/data-table/versions/1.7.0.um:4]: Yes, as I didn't want to copy the example again, I just added the existing. I want to retain the original examples and only add a new one

Comment (charlieTheBotDev) [reply content/modules/data-table/versions/1.7.0.um:4]: I probably should have been a bit more clear :)
At the top of the file there is `@version 1.5.0` but it should be `@version 1.7.0` otherwise `quantum-version` will do the wrong thing

Comment (jonthans) [reply content/modules/data-table/versions/1.7.0.um:4]: Oops. Well spotted

File: content/resources/hexagon/1.6.0/hexagon.core.css [hunk 2]
```diff
@@ -8,7 +8,7 @@
  
  ----------------------------------------------------
  
- Version: 1.6.0
```
Comment (jmsmyth) [inline content/resources/hexagon/1.6.0/hexagon.core.css:11]: It looks like the npm registry switching when installing hexagon went wrong somewhere. Probably best to revert these changes to the hexagon builds and start with a clean node_modules before doing the two npm installs?

Comment (charlieTheBotDev) [reply content/resources/hexagon/1.6.0/hexagon.core.css:11]: +1 - The changes in `content/resources` should only be commited when the new version of hexagon is released (another argument for merging the two repos :) )

**Comment by jmsmyth** [comment]
Merging into a branch in order to fix the hexagon versions
